manifests/params.pp

```diff
@@ -315,9 +334,17 @@
   $lower_fqdn              = downcase($::fqdn)
   $server_foreman          = true
   $server_facts            = true
-  $server_puppet_basedir   = $aio_package ? {
-    true  => '/opt/puppetlabs/puppet/lib/ruby/vendor_ruby/puppet',
-    false => undef,
+  case $::osfamily {
+    'Archlinux' : {
+      $server_puppet_basedir = '/usr/lib/ruby/vendor_ruby/2.3.0/puppet'
```

Comment:
if there's no other possible place for this, `$::rubyversion` should be used here